frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -23,14 +26,27 @@ const navigateTo = (e: Event, url: string) => {
   e.preventDefault();
 };
 
-const ODCEmptyState: React.FC<Props> = ({ title, activeNamespace }) => {
+const ODCEmptyState: React.FC<Props> = ({
+  title,
+  activeNamespace,
+  hintBlockTitle,
+  hintBlockDescription,
+}) => {
   return (
     <React.Fragment>
       <div className="odc-empty-state__title">
         <PageHeading title={title} />
-        <p className="co-catalog-page__description">
-          Select a way to create an application, component or service from one of the options.
-        </p>
+        {hintBlockTitle || hintBlockDescription ? (
```

Comment:
Can there be a description without the title?

Reply:
Title is not required prop in ```HintBlock ```. So, there can be a case where only the description is present.

Reply:
It can be `optional` in `HintBlock`  but that doesn't mean we will be having this case. It is a UX decision if we ever want to have hint block in empty state without the title and only the description. I think we will be having a title always so it is required for us.

Reply:
Remove the check for the description.